cinema/views.py

```diff
@@ -35,7 +41,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` are commented out. Ensure this is intentional, as it might leave the endpoints unprotected. Consider enabling them for production to secure the API.